### DISCUSSION TITLE
How add sidebar button?

### DISCUSSION THREAD

**Discussion by yablochnyi** [filamentphp/filament#3181] [Help]
Hello, can you tell me how to add a button to the sidebar? I need the buttons to perform actions from my controller and redirect to the same page.

And tell me please, in the database there are some fields with a value null, how to make the icon false be displayed?
![Снимок](https://user-images.githubusercontent.com/75489338/179953469-7a06d2ef-dce0-4cda-9c75-0812d708fb19.JPG)

**Comment by danharrin** [comment]
For the boolean column,

`->getStateUsing(fn ($record): bool => filled($record->description))`

**Comment by danharrin** [comment]
For the custom menu items, we have https://filamentphp.com/docs/2.x/admin/navigation#advanced-navigation-customization
